sql/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -333,4 +333,23 @@ object Connect {
       Option(System.getenv.get(CONNECT_AUTHENTICATE_TOKEN_ENV))
     }
   }
+
+  val CONNECT_SESSION_CONNECT_ML_CACHE_MAX_SIZE =
+    buildConf("spark.connect.session.connectML.mlCache.maxSize")
+      .doc("Maximum size of the MLCache per session. The cache will evict the least recently" +
+        "used models if the size exceeds this limit. The size is in bytes.")
+      .version("4.0.0")
```

Comment:
```suggestion
      .version("4.1.0")
```

I think this PR doesn't need to be included in 4.0.0